### ISSUE TITLE
AMP preview button seems to show duplicate logo when AMP sidebar is pinned

### ISSUE THREAD

**Issue by westonruter** [Automattic/amp-wp#5943]
## Bug Description

In Transitional/Reader modes, there is an AMP preview button that is separate from the regular Preview button. Additionally, as of #5589 there is a new plugin sidebar for displaying validation errors. When that sidebar is pinned, the result is an apparent duplication of the AMP buttons in the toolbar:

> ![image](https://user-images.githubusercontent.com/134745/110024756-575c0180-7ce3-11eb-8d22-294517a6b153.png)

Granted we want to eliminate the separate AMP preview button in favor of adding an AMP item to the Preview dropdown, but this is blocked by https://github.com/WordPress/gutenberg/issues/25309. Nevertheless, I wanted to point it out as a potential source of user confusion in the meantime.

The resolution for this issue may be `wontfix`.

## Expected Behaviour

The AMP logo should not appear multiple times in the toolbar.

## Steps to reproduce

1. Activate Reader or Transitional mode.
2. Edit a post.
2. Pin the AMP plugin sidebar
3. See two logos in the toolbar

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by simison** [comment]
There's a draft PR to add the capability in Gutenberg https://github.com/WordPress/gutenberg/pull/25430

@westonruter  Would anyone from the AMP team be up for helping to wrap that PR up? It seems so close! 😅

**Comment by westonruter** [comment]
@delawski Is https://github.com/WordPress/gutenberg/pull/25430 something you can take up?

**Comment by delawski** [comment]
@westonruter @simison I'll have a look and see what I can do about it.
